smoketests/tests/quickstart.py

```diff
@@ -309,6 +301,19 @@ def sdk_setup(self, path: Path):
             source_dir=(STDB_DIR / "crates/bindings-csharp/BSATN.Runtime").absolute(),
             build_subdir="bin/Release"
         )
+        # This one is only needed because the regression-tests subdir uses it
+        override_nuget_package(
+            project_dir=STDB_DIR/"sdks/csharp",
+            package="SpacetimeDB.Runtime",
+            source_dir=(STDB_DIR / "crates/bindings-csharp/Runtime").absolute(),
+            build_subdir="bin/Release"
+        )
+        override_nuget_package(
```

Comment:
Something similar is defined on like 298, is that intentional?

Reply:
Yes. One is an override for the C# SDK. This one overrides it for the temporary quickstart client we've created.